paasta_tools/firewall_update.py

```diff
@@ -95,16 +92,12 @@ def process_inotify_event(event, services_by_dependencies):
 
 
 def services_running_here(soa_dir):
-    # Generator helper that yields (service, instance, config) of both marathon and chronos tasks
-    cluster = load_system_paasta_config().get_cluster()
-
+    # Generator helper that yields (service, instance, framework) of both marathon and chronos tasks
```

Comment:
good catch -- I think I'll probably change the caller (seems less yucky since it's straightforward to fix the other call-site). just for context, I made this change because from the perspective of the cronjob, all of the rules it sets up are per (service, instance) pair as opposed to per running container, and it was somewhat awkward to gather them together before constructing the chain.